jboss/container/wildfly/launch/datasources/added/launch/datasource-common.sh

```diff
@@ -954,7 +954,7 @@ function inject_datasource() {
 
   if [ -z "$jta" ]; then
     log_warning "JTA flag not set, defaulting to true for datasource  ${service_name}"
-    jta=false
```

Comment:
@ochaloup  this change is unrelated to the issue. @kabir is it ok from your point of view?

Reply:
I think so, although I remember @yersan discussing this - let's wait for him to comment

Reply:
@kabir I discussed with Ondra only the details about the variables. Regarding the JTA change, assuming JTA by default to true is pretty reasonable in Java EE world, also it was the original intention regarding the logged message. This seems like a bug never reported.

Reply:
@ochaloup, we should separate this fix I think. Create a new CLOUD issue and a dedicated commit.

Reply:
@jfdenise sure, I see. I updated the PR and created a new one for this change: https://github.com/wildfly/wildfly-cekit-modules/pull/140